main/modes/utilities/ScreenSaver/screensaver.c

```diff
@@ -0,0 +1,356 @@
+/**
+ * @file screensaver.c
+ * @author Jeremy Stintzcum
+ * @brief Some swadge screensavers. Ruin your battery life
```

Comment:
Well at least you're honest about the battery here, and that's my biggest concern. I did some measurement, and this draws ~5x more current than LED idle (87.95 / 17.31 == 5.08, data below). I really don't want to give users the opportunity to demolish their battery and complain about it later. How do you feel about incorporating some kind of battery warning into the mode itself?

Screensaver (you can see the LED bounces:
![image](https://github.com/user-attachments/assets/21974dea-1a48-43b0-b73f-b0725345bffa)

LED idle (you can see the ESP-NOW TX & listen):
![image](https://github.com/user-attachments/assets/33d30261-f321-40a6-a1f2-a0997e3da749)

Reply:
Or like, don't call it a screensaver? Feels a little disingenuous. Icon Bounce?